src/palace/manager/customlists/explain.py

```diff
@@ -31,12 +31,6 @@ def _parse_arguments(args: list[str]) -> argparse.Namespace:
             default=0,
             help="Increase verbosity",
         )
-        parser.add_argument(
-            "--report-schema-file",
```

Comment:
Since these schema files are part of the package, it doesn't really make sense to pass them into the script anymore. Instead we load them from the package directly.